pkg/kv/bulk/sst_batcher.go

```diff
@@ -137,13 +137,14 @@ type SSTBatcher struct {
 
 	// The rest of the fields are per-batch and are reset via Reset() before each
 	// batch is started.
-	sstWriter       storage.SSTWriter
-	sstFile         *storage.MemFile
-	batchStartKey   []byte
-	batchEndKey     []byte
-	batchEndValue   []byte
-	flushKeyChecked bool
-	flushKey        roachpb.Key
+	sstWriter         storage.SSTWriter
+	sstFile           *storage.MemFile
+	batchStartKey     []byte
+	batchEndKey       []byte
+	batchEndValue     []byte
+	batchEndTimestamp hlc.Timestamp
```

Comment:
nit: at this point, we could just as well use a `storage.MVCCKeyValue` to gather all of these.